packages/example/pages/index.tsx

```diff
@@ -372,3 +385,14 @@ export default function Home(): React.ReactElement {
     </div>
   );
 }
+async function getBalances(
+  stableTokens: { symbol: string; contract: StableTokenWrapper }[],
```

Comment:
Everything looks good, but here is a small nitpick. I think `symbol` could be better than a `string`. We could maybe use the keys of `StableToken` as a type here

Reply:
good point